.github/workflows/release-php.yaml

```diff
@@ -0,0 +1,67 @@
+name: Release PHP
+on:
+  push:
+    tags:
+      - php/*
+jobs:
+  release-php:
+    strategy:
+      fail-fast: false
+      matrix:
+        versions:
+          - php: 7.3.33
+          - php: 7.4.32
+    runs-on: ubuntu-latest
+    env:
+      BINARYEN_VERSION: 110
+    steps:
+      - name: Checkout repository
+        # This if could be moved to the parent `job` section when it's
+        # supported by GitHub (https://github.com/community/community/discussions/37883)
+        if: github.event.ref == format('refs/tags/php/{0}', matrix.versions.php)
```

Comment:
I would add a comment to clarify this statement is required to build only the PHP version provided in the `tag`.